src/fastfield/multivalued/reader.rs

```diff
@@ -46,6 +46,24 @@ impl<Item: FastValue> MultiValuedFastFieldReader<Item> {
         self.vals_reader.get_range_u64(range.start, &mut vals[..]);
     }
 
+    /// Returns the minimum value for this fast field.
+    ///
+    /// The max value does not take in account of possible
```

Comment:
this comment seems like a copy-pasta

Reply:
Yes the comment looked like a warning that need to be clear to users from the internal reader. That's why I felt it's necessary to also have it on this methods.